### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by stavdlugo**

### CODE REVIEW

File: serrat/concerts.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+    <meta charset="UTF-8">
+    <title>Serrat</title>
+</head>
+<body>
+    <div id="nav"></div>
```
Comment (orelb) [inline serrat/concerts.html]: can you think of a better choice of element?

File: serrat/concerts.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            </section>
+            <section>
+                <p>Price: 67€, <a href="">Buy tickets</a></p>
+                <p>Date: 10/08</p>
+                <p>Location: San sebastian, Spain</p>
+            </section>
+        </article>
+    </main>
+
+    <div id="footer"></div>
```
Comment (orelb) [inline serrat/concerts.html]: find better choice of element

File: serrat/songs.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                <p>The song was written as a criticism of political corrupt leaders.</p>
+                <a href="https://www.youtube.com/watch?v=EXEEql6pnXY">Click here to listen</a>
+            </section>
+            <section>
+                <h4>Mediterráneo</h4>
+                <p>Serrat composes a song to the Mediterranean Sea, linking it to both his own life and the influence
+                    he exerts on the countries he bathes. According to an urban legend Serrat may have composed the song
+                    during his time locked in the Monastery of Montserrat,along with other artists,in protest at the
+                    Burgos Process in the late 1970s.</p>
+                <a href="https://www.youtube.com/watch?v=_w2WOHs9wG4">Click here to listen</a>
```
Comment (orelb) [inline serrat/songs.html:29]: Your links doesn't open in a new window.

**Review by orelb — CHANGES_REQUESTED** [review]

File: serrat/songs.html (hunk 3 above)
Comment (stavdlugo) [reply serrat/songs.html:29]: It was fixed with the addition of the yaml.

**Comment by orelb** [comment]
Good job :smile:

**Pull request merged**